lib/Target/SubstraitPB/Import.cpp

```diff
@@ -186,6 +208,9 @@ static mlir::FailureOr<RelOpInterface> importRel(ImplicitLocOpBuilder builder,
 
   Rel::RelTypeCase relType = message.rel_type_case();
   switch (relType) {
+  case Rel::RelTypeCase::kCross: {
+    return importCross(builder, message);
```

Comment:
For Read you used ReadRel, but Cross has no Rel suffix.

Reply:
Fixed.